pkg/builder/local_build_executor.go

```diff
@@ -152,6 +156,23 @@ func (be *localBuildExecutor) createOutputParentDirectory(inputRootDirectory fil
 	return d, nil
 }
 
+func (be *localBuildExecutor) createDev(inputRootDirectory BuildDirectory) error {
+	if err := inputRootDirectory.Mkdir("dev", 0777); err != nil && !os.IsExist(err) {
+		return util.StatusWrap(err, "Unable to create dev directory in input root")
```

Comment:
Even though the thing is called `dev`, maybe it's good to use `/dev` in error messages? That makes the intent a bit more clear.